te_markdown/define-password.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: "Tutorial: Define Passwords"
+permalink: /te_markdown/define-passwords/
+---
+
+Estimated reading time: 5 minutes
+
+## Motivation
+
+Often, applications under test require user credentials like username and password for user input. In order not to store user information directly in the test case descriptions in plain text, viewable for all test participants, the Test-Editor offers the possibility to make these data available to the test cases with following approaches. 
+
+
+* The tester can specify variables inside the test case (variables must exist as environment variables in the test environment).
+
+How to define these environment variables so that during the execution of test cases they can be used, is described in the following chapter.
+
+## How do I define credentials as environment variable?
+
+There are several ways to set an environment variable in the Test-Editor:
+
+* Definition of environment variables in the build.gradle file in the respective git user repository.
+* Definition of environment variables directly in the system where the tests should run (Jenkins, Docker, etc.)
+
+
+### Defining environment variables in build.gradle file 
+
+This following section should be included in the build.gradle file in the repository where the test cases are stored. In our Hero Tutorial for creating test cases [corresponding tutorial](/te_markdown/heroes-create-testcase) the repository with the build.gradle file resides at [GitHub](https://github.com/test-editor/language-examples/blob/tutorial/hero-create-testcase/build.gradle). After defining the variables in the build.gradle file, the environment variables are then available at start time of the test process. This possibility can be used in the initial creation of test cases, because the passwords defined in the build.gradle file can not be viewed by the users of the Test-Editor. However, it is possible that people who have access to the git repository where the testcases resides, can see the passwords in plain text in the build.gradle file. This circumstance is known and already been recorded as another TODO in the future. However, since the tests are mainly intended to run in the context of deployment pipelines, the variables should be stored in encrypted form in these systems where tests will be executed.
```

Comment:
"This following section" -> "The following section"

Reply:
"This possibility" -> "This method"

Reply:
"where the testcases resides," -> "where the testcases reside"

Reply:
"This circumstance is known and already been recorded as another TODO in the future." -> "This situation is known and has already been recorded as a future TODO."

Reply:
"encrypted form in these systems where tests will be executed" -> "encrypted form on the systems where tests are executed"